crates/runtime/src/config.rs

```diff
@@ -80,3 +97,57 @@ impl Default for Config {
         Self::new()
     }
 }
+
+#[cfg(feature = "cluster")]
+#[derive(Debug, Clone, clap::Parser)]
+pub struct ClusterConfig {
+    /// Configure cluster mode role
+    #[arg(
+        long = "cluster-mode",
```

Comment:
nit: We use `snake_case` for our params, but I think the convention is `kebab-case` for CLI apps, so that's why I checked this in. Wanted to bring it up, can move to snake if it's an issue.